MAUI/DataForm/layout.md

```diff
@@ -0,0 +1,428 @@
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Changing particular editors label position
+
+The label position of particular editor can be changed by using the [DataFormDefaultLayoutSettings.LabelPosition] property in `DataFormItem`, and it will be handled in the `GenerateDataFormItem` event.
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+             xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+    <dataForm:SfDataForm
+        x:Name="dataForm"
+        DataObject="{Binding DataObjects}"
+        GenerateDataFormItem="OnGenerateDataFormItem">
+    </dataForm:SfDataForm>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+        {
+            e.DataFormItem.DefaultLayoutSettings = new DataFormDefaultLayoutSettings
+            {
+                LabelPosition = DataFormLabelPosition.Left,
+            };
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Loading images for label
+
+Image can be loaded instead of label  by using [LeadingLabelIcon] property of `DataFormItem`.
+
+#### Using event
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "Name")
+            {
+                e.DataFormItem.LeadingLabelIcon = ImageSource.FromFile("ContactInfo.png");
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Changing order of the editors
+
+The order of the data form editors can be changed by using attributes or by handling `GenerateDataFormItem` event.
+
+#### Using attributes
+
+The data form editors order can be changed by using the `RowOrder` property in [DataFormDisplayOptionsAttribute] attribute.
+
+{% tabs %}
+{% highlight C# %}
+
+    [DataFormDisplayOptions(RowOrder = 1)]
+    public string Adress { get; set; }
+
+    [DataFormDisplayOptions(RowOrder = 0)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using event
+
+The data form editors order can be changed by using the [RowOrder] property in the `DataFormItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "Name")
+            {
+                e.DataFormItem.RowOrder = 0;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Changing editor order in grid rows
```

Comment:
Remove this as explained in Grouping 
## Changing editor order in grid rows

Reply:
Removed